test/testcmdlineparser.cpp

```diff
@@ -1411,6 +1430,25 @@ class TestCmdlineParser : public TestFixture {
         ASSERT_EQUALS("", settings->plistOutput);
     }
 
+    void outputFormatMixed() {
```

Comment:
Please separate this. Only a single usage is supposed to be done per test. Otherwise the data might not have been properly been reset between the calls.